app/javascript/notifications/notification.component.js

```diff
@@ -0,0 +1,84 @@
+// noinspection all
+
+
+const template = `
+
+<div class="card notification-message">
+  <div class="card-body">
+    <p class="card-title">
+      <strong>{{ type }}</strong> - {{ date }}, {{ time }}
+      <div class="pr-4">
+        <button @click="remove" class="text-right position-absolute top-0 end-0 text-primary fs-4 btn btn-icon" :id="'delete-notification-button-' + id">
+          <i class="bi bi-x"></i>
+        </button>
+      </div>
+    </p>
+    <p>{{ message }}</p>
+    <a class="card-link" @click="collapsed = !collapsed" v-if="expandable">
+        {{ collapsed ? "Show more" : "Show less" }}
```

Comment:
```suggestion
        {{ collapsed ? "\23F5" : "\23F6" }}
```

Entsprechend dem Stand auf dem Dev-Branch die Emojis nicht vergessen